### PR TITLE
fix(premeeting) Detach premeeting toolbar buttons visibility

### PR DESCRIPTION
**Pull request by horymury**
- if hiddenPremeetingButtons is undefined, toolbarButtons overwrite decides what buttons to show
- if hiddenPremeetingButtons is empty array, all buttons are show on premeeting screen regardless of toolbarButtons
- if hiddenPremeetingButtons hides some buttons, only those buttons will be hidden regardless of toolbarButtons overwrite

### CODE REVIEW

File: react/features/toolbox/components/web/Toolbox.js [hunk 1]
```diff
@@ -1430,14 +1430,7 @@ function _mapStateToProps(state, ownProps) {
...
 
-    let { toolbarButtons } = ownProps;
-    const stateToolbarButtons = getToolbarButtons(state);
-
-    if (toolbarButtons) {
-        toolbarButtons = toolbarButtons.filter(name => isToolbarButtonEnabled(name, stateToolbarButtons));
-    } else {
-        toolbarButtons = stateToolbarButtons;
-    }
+    const toolbarButtons = ownProps.toolbarButtons || getToolbarButtons(state);
```
Comment (saghul) [inline react/features/toolbox/components/web/Toolbox.js:1433]: Why is this change necessary?

**Review by saghul — APPROVED** [review]
One question, LGTM otherwise.

File: react/features/toolbox/components/web/Toolbox.js (hunk 1 above)
Comment (horymury) [reply react/features/toolbox/components/web/Toolbox.js:1433]: since we don't need an `if ... else` here anymore, thought to simplify the declaration.
